src/class-fes-preview-table.php

```diff
@@ -30,7 +30,7 @@ public function column_default( $item, $column_name ): string {
 				return isset( $item['content_type'] ) ? ucwords( implode( ', ', $item['content_type'] ) ) : __( 'Post, Page', 'wp-decoupled-preview' );
 			case 'actions':
 				return sprintf(
-					'<a href="%s">%s</a> | <a href="%s&TB_iframe=true&width=600&height=500" class="thickbox">%s</a> | <a href="%s&TB_iframe=true&width=600&height=500" class="thickbox">%s</a>',
+					'<a href="%s">%s</a> | <a href="%s&TB_iframe=true&width=600&height=500" class="thickbox">%s</a> | <a href="%s&TB_iframe=true&width=600&height=500" class="thickbox">%s</a> | <a href="%s&TB_iframe=true&width=600&height=500" class="thickbox">%s</a>',
```

Comment:
Should we keep using `iframe` for new actions or move to ajax?

Reply:
I think we should standardize on ajax at this point.